dirs/dirs_test.go

```diff
@@ -100,12 +99,7 @@ func (s *DirsTestSuite) TestClassicConfinementSupportOnSpecificDistributions(c *
 
 		// make a new root directory each time to isolate the test from
 		// local filesystem state and any previous test runs
-		altRoot, err := ioutil.TempDir("",
-			"TestClassicConfinementSupportOnSpecificDistributions")
-		c.Assert(err, IsNil)
-		defer os.RemoveAll(altRoot)
-
-		dirs.SetRootDir(altRoot)
+		dirs.SetRootDir(c.MkDir())
```

Comment:
Note that this is "persistent" so the suite or the test needs to undo that.

Reply:
There's a `defer dirs.SetRootDir("/")` at the beginning of this function.